src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Tab } from '@mui/material';
+import Tabs from '@mui/material/Tabs';
```

Comment:
I think it's use only one type of imports: named or default. Please, also check other places.
I've read about tree shaking in mui, and I understood that you can safely use named imports. Have you read about it?